models/labels/addresses/__single_category_labels__/worldcoin/labels_worldcoin_accounts.sql

```diff
@@ -0,0 +1,32 @@
+{{config(alias = alias('worldcoin_accounts'),
+        tags = ['dunesql'],
+        post_hook='{{ expose_spells(\'["optimism"]\',
+                                    "sector",
+                                    "labels",
+                                    \'["msilb7"]\') }}')
+}}
+
+{% set models = [
+    ('optimism', ref('worldcoin_optimism_accounts'))
+] %}
+{% for model in models %}
+
+SELECT
+    '{{ model[0] }}' as blockchain,
+    account_address as address,
+    'Worldcoin Account' AS name,
+    'worldcoin' AS category,
+    'msilb7' AS contributor,
+    'query' AS source,
+    from_iso8601_timestamp('2023-07-15') as created_at,
```

Comment:
`timestamp` function is supported now.
```suggestion
    timestamp '2023-07-15' as created_at,
```